client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -119,6 +119,7 @@ class JetpackChecklist extends PureComponent {
 							href={ JETPACK_CHECKLIST_TASK_AKISMET.getUrl( siteSlug ) }
 							inProgress={ ! akismetFinished }
 							onClick={ this.handleTaskStart( { taskId: 'jetpack_spam_filtering' } ) }
+							target="_blank"
```

Comment:
We'll also need to add `rel="noreferrer"`?

https://developers.google.com/web/tools/lighthouse/audits/noopener#recommendations

Reply:
The `target` prop [trickles down](https://github.com/Automattic/wp-calypso/blob/5d30badc44b3e794c71afa88f20138353458a1c9/client/components/checklist/task.js#L154) to the `Button` component, which [takes care of that](https://github.com/Automattic/wp-calypso/blob/5d30badc44b3e794c71afa88f20138353458a1c9/client/components/button/index.jsx#L38-L41) for us.

Reply:
#32853